ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -1057,13 +1057,14 @@ KEXT_STATIC bool ShouldIgnoreVnodeType(vtype vnodeType, vnode_t vnode)
 	case VREG:
     case VDIR:
     case VLNK:
-        return false;
+        break;
     case VSTR:
     case VCPLX:
         KextLog_FileInfo(vnode, "vnode with type %s encountered", vnodeType == VSTR ? "VSTR" : "VCPLX");
-        return false;
+        break;
     default:
         KextLog_FileInfo(vnode, "vnode with unknown type %d encountered", vnodeType);
-        return false;
     }
+    
+    return false;
```

Comment:
@nickgra XCode 9x's code coverage was confused by the previous function and thought the last 2 curly braces were never tested.  This was fixed in XCode10.

The force coverage for this function, I reworked it slightly.  I think its a fair tradeoff, although not something we want to do on a regular basis.